### ISSUE TITLE
How to align the ticks and cursor on a line when line and bar chart synchronized

### ISSUE THREAD

**Issue by claire1028** [recharts/recharts#3689]
- [ ] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Synchronized charts can ticks and cursor on a line when two graphics are laid out up and down,  as now synchronized charts just show the tooltip when has the same data index. 

### What does the proposed API look like?
isSyncTickLabel

**Comment by ckifer** [comment]
Hi @claire1028 the request here is hard to follow, can you clarify with an example of what you are trying to achieve?

**Comment by ckifer** [comment]
Going to close this as stale until some clarification is added, will re-open once it is
